search/search_repository.go

```diff
@@ -396,6 +399,12 @@ func (q Query) generateExpression() (criteria.Expression, error) {
 				} else {
 					myexpr = append(myexpr, criteria.Equals(left, right))
 				}
+			} else {
+				if child.Negate {
```

Comment:
I can see in tests as well that we are not supporting `!null`.
hmm.. that's good.

Reply:
Thanks :) If this feature is required in future, we can change the implementation.